src/pages/Autolending/AutolendingSettingsPage.vue

```diff
@@ -9,13 +9,7 @@
 		<autolending-who />
 
 		<div class="row column save-button-area">
-			<save-button class="show-for-large" />
```

Comment:
Why is the count and warning less important b/c you're using a mobile device?

Reply:
The count is now displayed in the "Who options" modal and also in the who you'll support section on the settings page. It is displayed the same on mobile and desktop. 

The warning is now also displayed the same on mobile and desktop (pop up modal message)